block_util/src/lib.rs

```diff
@@ -230,6 +232,7 @@ impl Request {
             status_addr: GuestAddress(0),
             writeback: true,
             aligned_operations: Vec::new(),
+            start: Instant::now(),
```

Comment:
You should make this an Option so that it's not started for every request when it's not needed.

Reply:
Thanks, I have updated it.